pages/onboarding_cluster.ipynb

```diff
@@ -0,0 +1,181 @@
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!opfcli create-project $NAMESPACE_NAME $TEAM_NAME -d $NAMESPACE_DISPLAY_NAME"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "##### Enabling namespace deployment to Zero cluster\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!cd cluster-scope/overlays/prod/$TARGET_CLUSTER"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Open the `kustomization.yaml` file in this folder and add `../../base/core/namespaces/$NAMESPACE` to the `resources` field. We aim to keep this field alphabetically sorted for better human readability:\n",
+    "```\n",
+    "---\n",
+    "apiVersion: kustomize.config.k8s.io/v1beta1\n",
+    "kind: Kustomization\n",
+    "\n",
+    "resources:\n",
+    "...\n",
+    "- ../../base/core/namespaces/$NAMESPACE\n",
+    "...\n",
+    "```\n",
+    "#### Populate your team\n",
+    "The following steps enable users to access designated cluster/namespaces. This consists of adding users to the appropriate OpenShift groups.\n",
+    "\n",
+    "Navigate to `cluster-scope/base/user.openshift.io/groups/$TEAM_NAME/`.\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!cd cluster-scope/base/user.openshift.io/groups/$TEAM_NAME/"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Edit group.yaml file by adding GitHub username of every user in the group.\n",
+    "```\n",
+    "---\n",
+    "apiVersion: user.openshift.io/v1\n",
+    "kind: Group\n",
+    "metadata:\n",
+    "  name: $TEAM_NAME\n",
+    "users:\n",
+    "  - user_1\n",
+    "  - user_2\n",
+    "```\n",
+    "```{note}\n",
+    "All usernames **must** be converted to lowercase!\n",
+    "```"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### Finalize\n",
+    "Please stage your changes and send them as a PR against the [operate-first/apps](https://github.com/operate-first/apps) repository. Make sure that following files/ have been modified/added:\n",
+    "- [x] `cluster-scope/base/core/namespaces/$NAMESPACE_NAME/kustomization.yaml`\n",
+    "- [x] `cluster-scope/base/core/namespaces/$NAMESPACE_NAME/namespace.yaml`\n",
+    "- [x] `cluster-scope/base/user.openshift.io/groups/$OWNER_TEAM/group.yaml`\n",
+    "- [x] `cluster-scope/base/user.openshift.io/groups/$OWNER_TEAM/kustomization.yaml`\n",
+    "- [x] `cluster-scope/components/namespace-admin-rolebinding/$OWNER_TEAM/kustomization.yaml`\n",
+    "- [x] `cluster-scope/components/namespace-admin-rolebinding/$OWNER_TEAM/rbac.yaml`\n",
+    "- [x] `cluster-scope/overlays/prod/$TARGET_CLUSTER/kustomization.yaml`\n",
+    "\n",
```

Comment:
Again, show it with output! Add a cmd cell with
```
git status
git add .
git commit -m ...
```

Reply:
Resolved in latest commit.